src/features/search/helpers/categoriesHelpers/mapping-tree.ts

```diff
@@ -6,16 +6,13 @@ import {
   SubcategoriesResponseModelv2,
 } from 'api/gen'
 import { CATEGORY_CRITERIA } from 'features/search/enums'
-import {
-  getFacetTypeFromGenreTypeKey,
-  getNativeCategories,
-} from 'features/search/helpers/categoriesHelpers/categoriesHelpers'
-import { FacetData, GenreTypeFacetData, NativeCategoryFacetData } from 'libs/algolia'
+import { getNativeCategories } from 'features/search/helpers/categoriesHelpers/categoriesHelpers'
+import { FacetData, NativeCategoryFacetData } from 'libs/algolia'
 import { FACETS_FILTERS_ENUM } from 'libs/algolia/enums'
 
 type MappedGenreType = {
   label: string
-  nbResultsFacet: number
+  nbResultsFacet?: never
 }
```

Comment:
Pourquoi ne pas simplement enlever la clé ?
```suggestion
type MappedGenreType = {
  label: string
}
```

Reply:
Ok c'est super bizarre... Autant qu'il dise que `item.nbResultsFacets` ne marche pas OK, mais `item.children` ???
Ok alors, soit, j'abdique

Reply:
Oui c'est vraiment bizarre que juste en retirant `nbResultsFacets` le type change et item est considéré seulement comme `MappedGenreType ` alors que data est bien typé